### PR TITLE
Added useToast()

### LINKED ISSUE #636: [Frontend] - Replace alert() with useToast() in AdminToolsUserManagement
### Description

Replace the alert() call with useToast() in AdminToolsUserManagement.

### Acceptance Criteria

The alert() call should be replaced by useToast() in AdminToolsUserManagement.

### Proposed Solution

`Ctrl + Shift + F`

Please refer to the notes in the [epic](https://github.com/Northeastern-Electric-Racing/FinishLine/issues/633) for more guidance.

### PR DESCRIPTION
**Pull request by AR619**
## Changes

imported toast, created a variable for it and called it when alert was called.

## Checklist

- [x] All commits are tagged with the ticket number
- [x] No linting errors / newline at end of file warnings
- [x] All code follows repository-configured prettier formatting
- [x] No merge conflicts
- [x] All checks passing
- [x] Screenshots of UI changes (see Screenshots section)
- [x] Remove any non-applicable sections of this template
- [x] Assign the PR to yourself
- [x] No `yarn.lock` changes (unless dependencies have changed)
- [x] Request reviewers & ping on Slack
- [x] PR is linked to the ticket (fill in the closes line below)

## Screenshot
Small SS
errorSmall

Big SS
bigss

Closes #636

### CODE REVIEW

File: src/frontend/src/pages/AdminToolsPage/AdminToolsUserManagement.tsx [hunk 1]
```diff
@@ -73,7 +75,10 @@ const AdminToolsUserMangaement: React.FC = () => {
       setHideSuccessLabel(false);
       setUser(null);
     } catch (e) {
-      alert(e);
+      if (e instanceof Error) {
+        alert(e.message);
```
Comment (Ari-S-123) [inline src/frontend/src/pages/AdminToolsPage/AdminToolsUserManagement.tsx:79]: You don't need this line.

**Review by Ari-S-123 — CHANGES_REQUESTED** [review]

File: src/frontend/src/pages/AdminToolsPage/AdminToolsUserManagement.tsx (hunk 1 above)
Comment (AR619) [reply src/frontend/src/pages/AdminToolsPage/AdminToolsUserManagement.tsx]: I've removed alert(e) and alert(e.message), is that fine or should I add alert(e) back? Cause in another ticket in this epic I saw that alert(e) was removed.

**Review by Ari-S-123 — APPROVED** [review]

**Review by Ari-S-123 — CHANGES_REQUESTED** [review]
Can you add a screenshot of the toast firing?

**Comment by AR619** [comment]
> Can you add a screenshot of the toast firing to the PR?

On it

**Comment by AR619** [comment]
> Can you add a screenshot of the toast firing to the PR?

done

**Review by Ari-S-123 — APPROVED** [review]

**Review by anthonybernardi — APPROVED** [review]
nice!

**Pull request merged**